### DISCUSSION TITLE
Recomendación de distro Linux

### DISCUSSION THREAD

**Discussion by StevieCL** [rramosp/ai4eng.v1#66] [00 - General]
Hola profe, le escribo por una duda que tengo referente a Linux: 
He visto en los vídeos que ud utiliza Linux en el pc, entonces me gustaría saber su opinión: Qué distro de Linux aconseja utilizar para Data science? Teniendo en cuenta que siempre he utilizado Windows. Muchas gracias!

**Comment by StevieCL** [comment]
pd: mi pc tiene un ryzen 5, 4 gb ram (pienso expandirla hasta 8 gb dentro de unos meses) y una tarjeta Radeon graphics.

**Comment by rramosp** [comment]
hola stevie ... yo suelo usar ubuntu, tanto en mi pc como en los
servidores, y me ha funcionado muy bien

El mar, 12 oct 2021 a las 0:10, StevieCL ***@***.***>)
escribió:

> Hola profe, le escribo por una duda que tengo referente a Linux:
> He visto en los vídeos que ud utiliza Linux en el pc, entonces me gustaría
> saber su opinión: Qué distro de Linux aconseja utilizar para Data science?
> Teniendo en cuenta que siempre he utilizado Windows. Muchas gracias!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/66>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIOXACACELCIQWFKEY3UGO7ODANCNFSM5FZVCYYA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by StevieCL** [reply]
Muchas gracias, profe!
